csrc/mmdeploy/backend_ops/onnxruntime/nms_match/nms_match.cpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) OpenMMLab. All rights reserved
+#include "nms_match.h"
+
+#include <assert.h>
+
+#include <algorithm>
+#include <cassert>
+#include <cmath>
+#include <iostream>
+#include <iterator>
+#include <numeric>
+#include <vector>
+
+#include "ort_utils.h"
+
+namespace mmdeploy {
+struct Box {
+    float x1, y1, x2, y2;
+};
+
+float nms_match_iou(Box box1, Box box2) {
+    auto max_x1 = std::max(box1.x1, box2.x1);
+    auto max_y1 = std::max(box1.y1, box2.y1);
+    auto max_x2 = std::min(box1.x2, box2.x2);
+    auto max_y2 = std::min(box1.y2, box2.y2);
+
+    auto w = std::max(static_cast<float>(0), max_x2 - max_x1);
+    auto h = std::max(static_cast<float>(0), max_y2 - max_y1);
+
+    auto area1 = (box1.x2 - box1.x1) * (box1.y2 - box1.y1);
+    auto area2 = (box1.x2 - box1.x1) * (box1.y2 - box1.y1);
```

Comment:
Should area2 computed with box2?

Reply:
Thanks for you review, I got that above.I will fix the code later

Reply:
@yinfan98 still not fixed?